### ISSUE TITLE
Encoding::UndefinedConversionError: "\x95" from ASCII-8BIT to UTF-8

### ISSUE THREAD

**Issue by cpunion** [CocoaPods/Xcodeproj#418]
I used xcodeproj-1.3.1 with xcode 8.0, ruby 2.3.1p112. `Dir.chdir` can't change working directory after I calls `Xcodeproj::Project#save`.

So I set `ENV['FORK_XCODE_WRITING'] = 'true'` to force it fork process (See https://github.com/CocoaPods/Xcodeproj/blob/master/lib/xcodeproj/plist/ffi.rb#L71), but the new problem is:
```
Encoding::UndefinedConversionError:
   "\x95" from ASCII-8BIT to UTF-8
# /Users/lijie/.rvm/gems/ruby-2.3.1/gems/xcodeproj-1.3.1/lib/xcodeproj/plist/ffi.rb:135:in `ruby_hash_write_xcode'
# /Users/lijie/.rvm/gems/ruby-2.3.1/gems/xcodeproj-1.3.1/lib/xcodeproj/plist/ffi.rb:72:in `write_to_path'
# /Users/lijie/.rvm/gems/ruby-2.3.1/gems/xcodeproj-1.3.1/lib/xcodeproj/plist.rb:46:in `write_to_path'
# /Users/lijie/.rvm/gems/ruby-2.3.1/gems/xcodeproj-1.3.1/lib/xcodeproj/project.rb:333:in `save'
```

Seems some marshaled value can't be pass to STDIN (See https://github.com/CocoaPods/Xcodeproj/blob/master/lib/xcodeproj/plist/ffi.rb#L135), I guess should encode it in an ASCII compatible format (e.g. Base64)?

**Comment by cpunion** [comment]
Tested Base64 and it passed, I will try to create a PR.

**Comment by cpunion** [comment]
https://github.com/CocoaPods/Xcodeproj/commit/04bdbae220fc59b1df2e832ffc2cc9810f74a25d fixed chdir, so don't need use `ENV['FORK_XCODE_WRITING'] = 'true'`.

**Comment by cpunion** [comment]
In version 1.3.3, `chdir` problem still occurs, so must set `ENV['FORK_XCODE_WRITING'] = 'true'`, and this issue reproduced.

**Comment by cpunion** [comment]
Fix in #419.

**Comment by cpunion** [comment]
Fixed by #430
